src/services/flightsDataService.js

```diff
@@ -9,9 +9,8 @@ const rootURL = "http://localhost:3030/";
 
 
 export const getFlights = async () => {
-    console.log('wooooo')
-    // const response = await axios.get(`${ rootURL }flights` );
-    return ['hello', 'world'];
+    const response = await axios.get(`${ rootURL }flights` );
+    return response.data;
```

Comment:
App will explode if axios throws an error

Reply:
Are you sure? Because I took down my BE server to test that this line in the App kicked in and it did... `{isError && <div className="flightsTable">Sorry, there was an error loading flights</div> }`

Reply:
@netia1128 It doesn't look like you actually use this provider anywhere (it doesn't wrap any components), so this code is not being invoked at all.

Reply:
Oh nevermind. I see you're importing `getFlights` into `App.jsx` which is really weird way to do things. And you're catching that there, so I get it. But yeah, kind of an anti-pattern there.

Reply:
Will also clean this up in the next PR.